app/controllers/songs_controller.rb

```diff
@@ -1,5 +1,34 @@
 class SongsController < ApplicationController
   def index
+    respond_to do |format|
```

Comment:
Exactly. Song data is grabbed asynchronously after the page loads.

On Tuesday, July 19, 2016, Patrick <notifications@github.com> wrote:

> In app/controllers/songs_controller.rb
> <https://github.com/GracepointMinistries/GraceTunes/pull/3#discussion_r71381730>
> :
>
> > @@ -1,5 +1,34 @@
> >  class SongsController < ApplicationController
> >    def index
> > +    respond_to do |format|
>
> I'm trying to understand the design and usage of the jQeury database gem.
> Correct me if I'm wrong: on the initial page load, Rails will not send any
> song data to the browser. But the JS on the page will get triggered and
> send a GET request and hit this code, which will then return all the songs?
> So the song data is never directly sent with the HTML body but retrieved
> when the gem requests it?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/GracepointMinistries/GraceTunes/pull/3/files/96c087ca3ff57c19187185fdbaf6649d2e0d54b0#r71381730>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFXKhFGJ-O1xcFeuBW1tpj3LirKtinpWks5qXQXigaJpZM4JP7Wk>
> .
>